ipsie-terminology.md

```diff
@@ -1,16 +1,18 @@
 # IPSIE Terminology
 
-### Enterprise / Enterprise Company
+## Roles
 
-Enterprises are generally defined as entities - e.g. corportations, non-profit organizations, partnerships.  Enterprise companies have a workforce comprised of employees, contractors, volunteers, and others who operate on behalf of the organization.  Enterprises deploy applications and services to support their organizational needs.  Government, non-governmental organizations, educational entities, and others may consider themselves enterprises.
+### Enterprise
+
+Enterprises are generally defined as entities - e.g. corportations, non-profit organizations, partnerships.  Enterprises have a workforce comprised of employees, contractors, volunteers, and others who operate on behalf of the enterprise.  Enterprises deploy applications and services to support their organizational needs.  Government, non-governmental organizations, educational entities, and others may consider themselves enterprises.
```

Comment:
Just adding this from our discussion on the weekly call:


**User as an Organization's Representative:**

- A user is recognized as a representative of an organization, empowered with certain privileges to procure and utilize services on the organization's behalf.

**Organizational Ownership and Rights:**

- The organization retains ownership over its user’s interactions with various service providers.

- The organization holds the authority to grant or revoke any individual user's privileges whenever deemed necessary.

**Verification of User Rights:**

- Before a user can act on behalf of their organization, their right to represent must be verified, beginning with the authentication of their identity.

**Delegation of Privileges:**

- Once an identity is authenticated, an organization can then delegate specific rights and responsibilities to that user.

**Cross-Organizational Trust:**

- An organization can opt to grant certain privileges or trust to users originating from outside its boundaries.

**Implementation of Policies:**

- Organizations implement procedures and decisions through well-defined policies and identity governance tools and frameworks.

Reply:
@ttripp so we don't lose this thread, maybe you can incorporate this as a PR into the terminology?